lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -41,7 +41,7 @@ def cluster_attributes cluster, datacenter_name
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{datacenter.name}|#{datacenter.hostFolder.name})\//,'')
+          cluster.pretty_path.gsub(/(#{datacenter_name}|#{datacenter.hostFolder.name})\//,'')
         end
       end
```

Comment:
Yes, but in <code>datacenter_name</code> is full path to DC which has to be removed from full path to cluster and only relative path from DC to cluster must remain.